connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -781,3 +782,10 @@ message FrameMap {
   CommonInlineUserDefinedFunction func = 2;
 }
 
+message WithSequenceColumn {
```

Comment:
We should really add a comment that this isn't supposed to be used by other external purposes at least.

Reply:
Just added the comment. Thanks!